### PR TITLE
cleaned up a little remove the ViewDecoratorMiddleware

### PR DESCRIPTION
**Pull request by kencochrane**
cleaned up a little remove the ViewDecoratorMiddleware

/cc @marcusmartins @shin- please review.

### CODE REVIEW

File: defender/tests.py [hunk 1]
```diff
@@ -501,7 +501,6 @@ def test_admin(self):
         from .admin import AccessAttemptAdmin
         AccessAttemptAdmin
 
-    @patch('defender.config.PROTECTED_LOGINS', (ADMIN_LOGIN_URL, ))
     def test_decorator_middleware(self):
```
Comment (marcusmartins) [inline defender/tests.py]: @kencochrane Is this test still necessary?

**Comment by marcusmartins** [comment]
@kencochrane Made a comment other than that It looks good to me.

File: defender/tests.py (hunk 1 above)
Comment (kencochrane) [inline defender/tests.py]: I'm not sure @shin- added it I think. I think it tests the other middleware still, but I could be wrong. 

@shin- what do you think?

Comment (shin-) [inline defender/tests.py]: if we removed the middleware we don't need the middleware test anymore :)

Comment (kencochrane) [inline defender/tests.py]: @shin- there is still one of the middlewares. there was 2 originally, then I removed one. so there is still 1 left

**Comment by kencochrane** [comment]
resolves #23

**Comment by marcusmartins** [comment]
LGTM

**Pull request merged**